cmd/metal-api/internal/service/v1/machine.go

```diff
@@ -103,6 +103,8 @@ type MachineRecentProvisioningEvents struct {
 	Events                       []MachineProvisioningEvent `json:"log" description:"the log of recent machine provisioning events"`
 	LastEventTime                *time.Time                 `json:"last_event_time" description:"the time where the last event was received" optional:"true"`
 	IncompleteProvisioningCycles string                     `json:"incomplete_provisioning_cycles" description:"the amount of incomplete provisioning cycles in the event container"`
```

Comment:
I will check if there is a field annotation available that marks this deprecated. If it's not possible, just put into the description.